pkg/util/cluster/client.go

```diff
@@ -0,0 +1,164 @@
+/*
+Copyright 2023 The KubeAdmiral Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	fedcorev1a1 "github.com/kubewharf/kubeadmiral/pkg/apis/core/v1alpha1"
+)
+
+// User account keys
+const (
+	ClientCertificateKey    = "client-certificate-data"
+	ClientKeyKey            = "client-key-data"
+	CertificateAuthorityKey = "certificate-authority-data"
+)
+
+// Service account keys
+const (
+	ServiceAccountTokenKey = "service-account-token-data"
+	ServiceAccountCAKey    = "service-account-ca-data"
+)
+
+func BuildClusterConfig(
```

Comment:
Are these copied without modification? What happens to the original file?

Reply:
Yes, I should have deleted the original file.